src/polymerday-sponsors-list.html

```diff
@@ -1,32 +1,72 @@
 <link rel="import" href="../bower_components/polymer/polymer.html">
-
 <link rel="import" href="polymerday-data.html">
+<link rel="import" href="polymerday-sponsor-item.html">
 
 <dom-module id="polymerday-sponsors-list">
-
   <template>
 
-  <style>
-
-    a {
-      text-decoration: none;
-    }
-  </style>
-
-  <polymerday-data
-    page="sponsors"
-    subpage="{{sponsorsType}}"
-    data="{{sponsors}}"></polymerday-data>
-
-  <template is="dom-if" if="[[sponsors.length]]">
-    <polymerday-section-title title="[[sponsorsTitle]]"></polymerday-section-title>
-    <template is="dom-repeat" items="{{sponsors}}" as="sponsor">
-        <a href="[[sponsor.link]]">
-          <img src="[[sponsor.logo]]" alt="[[sponsor.name]]"/>
-        </a>
-    </template>
-  </template>
+    <style>
+
+      .sponsors--container {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: flex-start;
+      }
+
+      .sponsor-platinum {
+        width: calc(100% / 2);
+      }
+
+      .sponsor-gold {
+        width: calc(100% / 3);
+      }
+
+      .sponsor-iron {
+        width: calc(100% / 4);
+      }
+
+      .sponsor-paper {
+        width: calc(100% / 8);
+      }
+
+      @media (max-width:767px) {
+
+        .sponsors--container--gold,
+        .sponsors--container--paper {
+          justify-content: center;
+        }
 
+        .sponsor-platinum {
+          width: 100%;
+        }
+
+        .sponsor-gold {
+          width: 80%;
+        }
+
+        .sponsor-iron {
+          width: calc(100% / 2);
+        }
+
+        .sponsor-paper {
+          width: calc(90% / 2);
+        }
+      }
+
+    </style>
+
+    <polymerday-data
```

Comment:
Sorry about that, I guess it's better to take out this and avoid to create a dependency with the data access.